transforms/__testfixtures__/touchable.output.js

```diff
@@ -2,9 +2,9 @@
 
 <View>
   <TouchableBounce style={{color: 'red'}}>
-    <Image />
-    <Image />
-  </TouchableBounce>
+          <Image />
+          <Image />
+        </TouchableBounce>
```

Comment:
can you revert this change? This is a bug in recast that still hasn't been reverted :( It's ok to keep this test failing. We can also just remove it, the touchable codemod isn't relevant any more.